marconi-sidechain/src/Marconi/Sidechain/Bootstrap.hs

```diff
@@ -41,27 +54,28 @@ import System.FilePath ((</>))
 -- | Run Sidechain indexers
 runSidechainIndexers :: ReaderT SidechainEnv IO ()
 runSidechainIndexers = do
-  env <- ask
-  cliArgs <- view sidechainCliArgs
+  CliArgs{..} <- view sidechainCliArgs
   trace <- view sidechainTrace
+  env <- view sidechainIndexersEnv
+
   let addressUtxoCallback :: State UtxoHandle -> IO ()
       addressUtxoCallback =
         atomically
-          . AddressUtxo.updateEnvState (env ^. sidechainIndexersEnv . sidechainAddressUtxoIndexer)
+          . AddressUtxo.updateEnvState (env ^. sidechainAddressUtxoIndexer)
   let epochStateCallback :: State EpochStateHandle -> IO ()
       epochStateCallback =
         atomically
           . EpochState.updateEnvState
-            (env ^. sidechainIndexersEnv . sidechainEpochStateIndexer . epochStateIndexerEnvIndexer)
+            (env ^. sidechainEpochStateIndexer . epochStateIndexerEnvIndexer)
   let mintBurnCallback :: State MintBurnHandle -> IO ()
-      dbPath = CLI.dbDir cliArgs
+      dbPath = dbDir
```

Comment:
Remove?

Reply:
Good call - thanks. Forgot to make that change.